### ISSUE TITLE
NTR: N-acetylglucosamine-containing oligosaccharides in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8818]
New term request N-acetylglucosamine-containing oligosaccharides in urine (segal_070925154923):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000122
- label: Increased urinary N-acetylglucosamine-rich oligosaccharide level
- parents: [Oligosacchariduria (HP:0010471)](https://hpo.jax.org/app/browse/term/HP:0010471)
- pmid: 
- def: Presence of N-acetylglucosamine-rich oligosaccharides in the urine.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8818
- synonyms:
